MethodDecorator/MethodDecorator.csproj

```diff
@@ -7,7 +7,7 @@
     <Authors>Simon Cropp, Matt Ellis, Alexey Suvorov, Andy Hoyle, Tony (@megafinz), and contributors</Authors>
     <Description>Fody add-in to decorate arbitrary methods to run code before and after invocation.</Description>
     <PackageTags>ILWeaving Fody AOP</PackageTags>
-    <NoWarn>CS1591;NU5228</NoWarn>
+    <NoWarn>CS1591;NU5228;NU5118</NoWarn>
```

Comment:
Doesn't that hide an issue?

Reply:
Yes, so this warning is actually defined in `Directory.Build.props`. Because the project is set to treat all warnings as errors, when the weaving happens for the generated dll, it causes it to error out because the dll already exists.

What we want is it to ignore the warning and override the dll before packaging. 

At least that's my understanding. 

When I run the build in Rider or Command Line, it throws NU5118, if it's in Visual Studio on Windows it doesn't appear to throw as I believe it picks up the setting from `Directory.Build.props`